src/administration/data-retention.md

```diff
@@ -0,0 +1,39 @@
+# Data retention
+
+Platform.sh logs and stores all sorts of data as a normal part of its business.  This information is retained as needed for business purposes and old data is purged.  The retention time varies depending on the type of data stored.
+
+## Account information
+
+Information relating to customer accounts (login information, billing information, etc.) is retained for as long as the account is active with Platform.sh.
+
+Customers may request that their account be deleted and all related data be purged by filing an issue ticket.
+
+## System logs
+
+System level access and security logs are maintained by Platform.sh for diagnostic purposes.  These logs are not customer-accessible.  These logs are retained for at least _6 months_ and at most _1 year_.
+
+General system level logs are retained for at least _30 days_ and at most _1 year_.
+
+## Payment processing logs
+
+Logs related to payment processing are retained for at least _3 months_ and at most _1 year_.  This is consistent with PCI recommendations.
+
+## Application logs
+
+Application logs on each customer environment are retained with the environment.  Individual log files are truncated at 10 MB, regardless of their age.  See the [accessing logs](/development/logs.md) page for instructions on how to access them.
+
+When an environment is deleted its application logs are deleted as well.
+
+## Snapshots
+
+Snapshots on Platform.sh Professional are retained for at least _7 days_.  They will be purged after X days.
```

Comment:
@ralt @rinchen When do we want to say snapshots get purged for reals?

Reply:
"between 7 days and 6 months"